### PR TITLE
correct doc for DF.std

### PR DESCRIPTION
**Pull request by Rubtsowa**

### CODE REVIEW

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 1]
```diff
@@ -715,14 +715,14 @@ def std_overload(df, axis=None, skipna=None, level=None, ddof=1, numeric_only=No
 
     Limitations
     -----------
-    Parameters axis, skipna, level, ddof, numeric_only unsupported.
+    Parameters axis, skipna, level, ddof, numeric_only are unsupported.
```
Comment (PokhodenkoSA) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]: Please, highlight parameter names and use **and** between the last two:
```suggestion
    Parameters ``axis``, skipna, level, ddof and numeric_only are unsupported.
```

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 2]
```diff
@@ -803,7 +803,7 @@ def std_overload(df, axis=None, skipna=None, level=None, ddof=1, numeric_only=No
 
     Limitations
     -----------
-    Parameters axis, skipna, level, ddof, numeric_only are unsupported.
+    Parameters ``axis``, ``skipna``, ``level``, ``ddof`` and ``numeric_only`` are unsupported.
```
Comment (PokhodenkoSA) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]: Is `skipna` really unsupported?

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 3]
```diff
@@ -796,32 +796,35 @@ def mean_overload(df, axis=None, skipna=None, level=None, numeric_only=None):
...
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
 
-       .. only:: developer
+    Pandas API: pandas.DataFrame.std
 
-           Test: python -m sdc.runtests -k sdc.tests.test_dataframe.TestDataFrame.test_std*
+    Limitations
+    -----------
+    Parameters ``axis``, ``level``, ``ddof`` and ``numeric_only`` are unsupported.
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]: Actually parameter `ddof` is supported.

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 4]
```diff
@@ -800,6 +800,14 @@ def std_overload(df, axis=None, skipna=None, level=None, ddof=1, numeric_only=No
     .. command-output:: python ./dataframe/dataframe_std.py
        :cwd: ../../../examples
 
+    .. seealso::
+        :ref:`Series.std <pandas.Series.std>`
+            Returns sample standard deviation over Series.
+        :ref:`Series.var<pandas.Series.var>`
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]:
```suggestion
        :ref:`Series.var <pandas.Series.var>`
```

**Pull request merged**